src/api/routers/sitesRouter.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+
+import { isApproverCheck } from '../middleware/approversMiddleware';
+import { getSiteList } from '../services/adminServiceClient';
+
+export function createSitesRouter() {
+  const sitesRouter = express.Router();
+
+  sitesRouter.use('/', isApproverCheck);
+
+  sitesRouter.get('/', async (req, res) => {
```

Comment:
is it worth prefixing `req` with an underscore since it is not used?

Reply:
Fixed